src/psyclone/dynamo0p3.py

```diff
@@ -7871,14 +8038,33 @@ def ref_element_properties(self):
                         [str(sprop) for sprop in RefElementMetaData.Property]))
 
     def quad_rule(self):
```

Comment:
P.S I think it would be good to declare the `quad_rule` in the base class `ArgOrdering` as abstract - we would not need the `NotImplementedError` there any more.